admin/src/templates/edit_translation.html

```diff
@@ -1,7 +1,7 @@
 <mm-modal
   status="status"
   id="'edit-translation'"
-  title-key="editing ? 'Edit translation' : 'translation.add'"
+  title-key="editing ? 'Edit translation' : 'translation.add.key'"
```

Comment:
I think we should have the button and the modal dialog title text match "Add new translation key". That means we can just use the one key, so leave this one unchanged and just update the existing value.